### PR TITLE
cakebrewjs 2.0.0

### PR DESCRIPTION
**Pull request by shemeshg**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/c/cakebrewjs.rb [hunk 1]
```diff
@@ -20,5 +20,6 @@
     "~/Library/Caches/cakebrewjs",
     "~/Library/Preferences/com.electron.cakebrewjs.helper.plist",
     "~/Library/Preferences/com.electron.cakebrewjs.plist",
```
Comment (chenrui333) [inline Casks/c/cakebrewjs.rb:22]: these two should be removed?

Comment (shemeshg) [reply Casks/c/cakebrewjs.rb:22]: correct, these are version 1 plists.

Comment (chenrui333) [reply Casks/c/cakebrewjs.rb:22]: so maybe remove them?

Comment (chenrui333) [reply Casks/c/cakebrewjs.rb:22]: oh the thoughts in here is for the upgrade? :hmm:

Comment (shemeshg) [reply Casks/c/cakebrewjs.rb:22]: unless we have conditional ZAP ...

Comment (krehel) [reply Casks/c/cakebrewjs.rb:22]: This isn't a big deal IMHO.  if a user upgrades and then deletes the application, we should cleanup the old files.  If this is a new install and the files don't exist, then no harm.

Comment (chenrui333) [reply Casks/c/cakebrewjs.rb:22]: 👍

**Review by chenrui333 — APPROVED** [review]

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**